modules/tinymce/src/plugins/help/test/ts/module/Selectors.ts

```diff
@@ -1,5 +1,3 @@
-export default {
-  dialog: '[role="dialog"]',
-  toolbarHelpButton: 'button',
-  pluginsTab: '[role="tab"]:contains(Plugins)'
-};
+export const dialog = '[role="dialog"]';
```

Comment:
This probably could just export a single const, as it's sort of weird to be exporting them individually 🤷‍♀️

Reply:
We actually just export `selectors` elsewhere as well: https://github.com/tinymce/tinymce/blob/6ea2a4f21bf2c582a62febdea9c1f8b535580ebe/modules/tinymce/src/plugins/media/test/ts/module/test/Utils.ts#L9 so I think we should change this do something similar?

Reply:
Done. Does this look better?

Reply:
Looks good, thanks! :+1: